element_array_ephys/export/nwb/tests.py

```diff
@@ -0,0 +1,37 @@
+from element_array_ephys.export.nwb.nwb import ecephys_session_to_nwb, write_nwb
+
+from pynwb.ecephys import ElectricalSeries
+
+
+def test_convert_to_nwb():
+
+    nwbfile = ecephys_session_to_nwb(
+        dict(subject="subject5", session_datetime="2020-05-12 04:13:07")
```

Comment:
My local version of this file doesn't match what's on the NWB export server. Instead, mine was built with `workflow-array-ephys` whose `ingest.py` [pulls from the SpikeGLX file itself](https://github.com/datajoint/workflow-array-ephys/blob/main/workflow_array_ephys/ingest.py#L58)
```
dict(subject="subject5", session_datetime="2018-07-03 20:32:28",lfp="dj")
```

Reply:
I don't know. I suggest you resolve the conflict?

Reply:
@CBroz1 lfp="dj" is an input arg, but it shouldn't go in this dict. See the new test below:  

```python
nwbfile = ecephys_session_to_nwb(
        dict(subject="subject5", session_datetime="2020-05-12 04:13:07"),
        lfp="dj",
        spikes=False,
    )
```